### ISSUE TITLE
merged of temporary databases fail assertion  with multible pulsemaps

### ISSUE THREAD

**Issue by MoustHolmes** [graphnet-team/graphnet#362]
**Describe the bug**
When running convert_i3_files the sqlite_dataconverter fails at a assertion in _extract_table_names() at  [line 116](https://github.com/graphnet-team/graphnet/blob/main/src/graphnet/data/sqlite/sqlite_dataconverter.py#L116) 
`assert all([results[0] == r for r in results])` 
 after digging around I found that the table names don't always come out in the same order in the list, therefore, failing the assertion.

a quick fix was to change the assertion to this `assert all([sorted(results[0]) == sorted(r) for r in results])`

```
first list of table names
('truth', 'retro', 'SplitInIcePulses', 'SplitInIcePulsesSRT', 'SplitInIcePulsesCleaned', 'SplitInIcePulsesTWSRT', 'SplitInIcePulses_GraphSage_Pulses', 'I3MCTree__primaries', 'I3MCTree__particles', 'I3TriggerHierarchy')

first failed list of table names
('truth', 'retro', 'SplitInIcePulses', 'SplitInIcePulsesSRT', 'SplitInIcePulsesCleaned', 'SplitInIcePulses_GraphSage_Pulses', 'I3MCTree__primaries', 'I3MCTree__particles', 'I3TriggerHierarchy', 'SplitInIcePulsesTWSRT')

Traceback (most recent call last):
  File "/home/moust/work/nmo_analysis/convert_i3_files.py", line 108, in <module>
    main_icecube_upgrade(backend)
  File "/home/moust/work/nmo_analysis/convert_i3_files.py", line 101, in main_icecube_upgrade
    converter.merge_files(os.path.join(outdir, "merged_140021"),input_files)
  File "/home/moust/work/graphnet/src/graphnet/data/sqlite/sqlite_dataconverter.py", line 77, in merge_files
    table_names = self._extract_table_names(input_files)
  File "/home/moust/work/graphnet/src/graphnet/data/sqlite/sqlite_dataconverter.py", line 106, in _extract_table_names
    assert all([results[0] == r for r in results])
AssertionError
```

**Comment by asogaard** [comment]
Hi @MoustHolmes,
This looks an awful lot like #336 which was fixed by #342. First off, could you confirm that you are working on a branch that includes this latter PR?

**Comment by RasmusOrsoe** [comment]
@asogaard I second this. The issue is fixed.
